ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/parameter/AbstractParameterMapper.java

```diff
@@ -55,8 +59,15 @@ public void setParameter() throws ParameterMapperException {
         parameterList.forEach(operationInventory::setParameter);
     }
 
-    static Object getDefaultValue(DefaultableParameterNode parameterNode) {
-        Node defaultValueExpression = parameterNode.expression();
+    static Object getDefaultValue(DefaultableParameterNode parameterNode, SemanticModel semanticModel) {
+        ExpressionNode defaultValueExpression = (ExpressionNode) parameterNode.expression();
+        Optional<Symbol> symbol = semanticModel.symbol(defaultValueExpression);
+        if (symbol.isPresent() && symbol.get() instanceof ConstantSymbol constantSymbol) {
+            Object constValue = constantSymbol.constValue();
+            if (constValue instanceof ConstantValue value) {
+                return value.value();
+            }
+        }
```

Comment:
Shall we separate this as a common util function to extract the constant value from node using the semantic model. It may be useful in the future